### PR TITLE
Add note on changing node IP

### PR DESCRIPTION
**Pull request by cstrangedk**

### CODE REVIEW

File: validators/faq.adoc [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+http://skale.chat/[Ask a question]
+
+== Validator Node
+
+=== How do I change my Node's IP address?
+
+It would be best to avoid having to change your node's IP address. It's recommended to use a permanent IP that can dynamically point to a node's machine IP (for example, elastic IP) so you can update machines if needed without having to change the IP recorded in SKALE Manager during node registration.
+
+If you must change your node IP in SKALE Manager, you must perform a node exit that could be costly to the node operator if it's currently supporting SKALE Chains.
```
Comment (ebruengwall-ant) [inline validators/faq.adoc]: We have to mention that node exit and re-registration with the new node IP has to be completed the first 3 days of each month. And ends on the 3rd day of the month UTC midnight.

File: validators/faq.adoc [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+To perform a node exit, follow these general steps:
+
+1. Execute `skale exit start`.
+2. Verify the exit process by executing `skale exit status`. (You don't need to wait for exit to finalize)
+3. Register a new node with the new IP.
```
Comment (ebruengwall-ant) [inline validators/faq.adoc]: link node to the validator address and register a new node with the new IP

**Review by ebruengwall-ant — DISMISSED** [review]
Looks good and please add the sentence I requested

**Review by ebruengwall-ant — DISMISSED** [review]

File: validators/faq.adoc [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+
+
+
+Need help solving an issue? Check to see if this has already been answered below. If you can't find an answer to your issue, reach out to the SKALE Network developer community on discord.  
+
+http://skale.chat/[Ask a question]
+
+== Validator Node
+
+=== How do I change my Node's IP address?
```
Comment (dmytrotkk) [inline validators/faq.adoc:25]: I think this title and FAQ overall is a bit confusing. Cause in reality you COULDN'T change node IP address. What we're describing here it's basically a new node registration, just with keeping the old SGX key.

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: validators/faq.adoc [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+=== Where are the network communications between containers, shared configuration files and volumes stored?
+
+You can find these details in GitHub [skale-node](https://github.com/skalenetwork/skale-node) repository and [docker-compose.yml](https://github.com/skalenetwork/skale-node/blob/f928b95e69c548f12b4b21bd11a16fe2d239b83b/docker-compose.yml)  
+
+=== How do I access the SKALE Chain's IP and ports?
+
+Please perform the following in your terminal and pick one of the names shows up under "schains" folder:  
+
+```shell
```
Comment (badrogger) [inline validators/faq.adoc:67]: It's not quite correct. This is the path of the mounted directory inside container. To get needed data from host you can check ~/.skale/node_data/schains/schain_schain_name.json file or use `skale schains show-rules` to show firewall rules info that contains IP and ports.

**Review by badrogger — CHANGES_REQUESTED** [review]

File: validators/faq.adoc (hunk 4 above)
Comment (dmytrotkk) [reply validators/faq.adoc:67]: +1, the path is not correct.

Also, config file could be retrieved using node-cli, without going into internal folders:
```bash
skale schains config SCHAIN_NAME
```

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: validators/faq.adoc [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+docker logs runsgx_sgxwallet_1
+```
+
+node-cli debugging logs
+
+```shell
+~/.skale/.skale-cli-log/debug-node-cli.log
+```
```
[the remark this answers is not in the document]
Comment (cstrangedk) [reply validators/faq.adoc:158]: added

File: validators/faq.adoc [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+transaction manager logs
+
+```shell
+docker logs skale_transaction_manager
+```
+
+all logs
+
+```shell
+skale logs dump [PATH]
```
[the remark this answers is not in the document]
Comment (cstrangedk) [reply validators/faq.adoc:139]: done

File: validators/faq.adoc (hunk 3 above)
Comment (cstrangedk) [reply validators/faq.adoc:25]: done

File: validators/faq.adoc (hunk 4 above)
Comment (cstrangedk) [reply validators/faq.adoc:67]: done

**Review by gannakulikova — APPROVED** [review]

**Review by badrogger — APPROVED** [review]

**Pull request merged**